spec/01-lexical-syntax.md

```diff
@@ -553,6 +553,22 @@ but are required to be properly nested.  Therefore, a comment like
 `/* /* */` will be rejected as having an unterminated
 comment.
 
+## Multi-line and trailing commas
+
+Trailing commas are allowed when followed by a newline and a closing
```

Comment:
> I suggest bringing this closer to the implementation, which ignores a (single) comma if the following token is a right-parenthesish that itself comes after a line end.

I can certainly write what you wrote. But I don't understand what difference(s) you're highlighting. Is it the "ignores"? "newline" vs "line end"?

---

> By the way, when looking at the implementation again, I noticed that the only other use of `afterLineEnd` is predicated on `!applyBracePatch` (presumably for the IDE). Should we do that here too?

I had a look at `applyBracePatch` by I can't grok what it does. If you can provide some more context, or even better a test case where you expect the current logic to fall over I'd be happy to look into it.